draft-ietf-quic-qpack.md

```diff
@@ -262,8 +262,8 @@ Indexed-Duplicate representation instead (see {{indexed-duplicate}}).
 an existing entry. {{RFC7541}} allows duplicate HPACK table entries, that is
 entries that have the same name and value.
 
-This replaces the HPACK instruction for Dynamic Table Size Update (see Section
-6.3 of {{RFC7541}}, which is not supported by HTTP over QUIC.
+This repurposes the HPACK instruction for Indexed Header Field (see Section
+6.1 of {{RFC7541}}.
```

Comment:
This wording might be confusing to some readers.  It might be more accurate to say that you're defining the behavior of this instruction on the control stream, while it retains its usual meaning on the request streams.